e2e/libs/node/aws.py

```diff
@@ -0,0 +1,62 @@
+import boto3
+import logging
+
+class EC2:
+
+    _ec2_instance = boto3.resource('ec2')
+    _all_nodes = None
+
+    def __init__(self, all_nodes) -> None:
+        super().__init__()
+        self._all_nodes = all_nodes
+
+    def get_all_node_instances(self):
```

Comment:
It will throw an exception if there is a problem getting/starting/stopping the ec2 instance operation. I've updated it to catch and logging the exception message. 
https://github.com/longhorn/longhorn-tests/blob/ffa1571e24818a52b2d1f4bd55bae3e9184bab43/e2e/libs/node/aws.py#L45-L49

Reply:
If it will throw an exception, how long it will wait before timeout and throw the exception?
It's highly related to how long our test will spend on waiting and the total execution time of a full test. 
Can you help to try running this test case without setting any aws related config?

Reply:
I'd tried two scenarios and found it will raise exception immediately.
 - missing credential file
![image](https://github.com/longhorn/longhorn-tests/assets/17548901/b5f6f41b-6818-40c6-a226-5c38b823e370)
 - invalid access credential
 ![image](https://github.com/longhorn/longhorn-tests/assets/17548901/a3167ac1-96a1-401b-8fdf-0f3cf95c696a)